interceptors/event-listener-interceptors/src/main/java/uk/gov/justice/services/components/event/listener/interceptors/EventFilterInterceptor.java

```diff
@@ -15,8 +15,10 @@
 
     @Override
```

Comment:
Isn't this a generated class now, with the specific EventFilter class not injected?

Reply:
This can be removed...but probably should be deprecated first...I did deprecate in another pull request to support multiple topics

Reply:
Sounds good